src/components/Switch/Switch.tsx

```diff
@@ -0,0 +1,148 @@
+import React, {useMemo} from 'react'
+import Animated, {useAnimatedStyle, withTiming} from 'react-native-reanimated'
+import styled from 'styled-components/native'
+import {theme as defaultTheme} from '../../theme'
+import type {ISwitchContainer, ILabel, IThumb, ITrack, ITrackSize, SwitchProps} from './switch.types'
+import {
+  SPACING_INSIDE,
+  SPACING_OUTSIDE,
+  SwitchVariant,
+  TEXT_INSIDE,
+  TEXT_INSIDE_COLOR,
+  THUMB_SIZE,
+  TRACK_RADIUS,
+  TRACK_WIDTH,
+} from './switch.utils'
```

Comment:
I believe it is more likely the constants file , not the utils file.

Reply:
updated